### PR TITLE
Fix sim hopper

### PR DESCRIPTION
**Pull request by rsinnet**
It is broken on develop due to some recent changes.

### CODE REVIEW

**Review by jordanra-miso — APPROVED** [review]

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py [hunk 1]
```diff
@@ -92,7 +92,9 @@ class SimSingleHopperDriver(SingleHopperDriver):
 
     def __init__(self,  dispenser, hopper_parameters):
         """Construct a new SimSingleHopperDriver."""
-        super(SimSingleHopperDriver, self).__init__(hopper_parameters)
+        super(SimSingleHopperDriver, self).__init__(
+            hopper_params['portion_sizes'],
```
Comment (ludwig) [inline miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:96]: Should be `hopper_parameters`? Or rename argument name, I suppose.

Comment (rsinnet) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:96]: hmmmmm

**Comment by rsinnet** [comment]
I'll test before merging for sure.

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py (hunk 1 above)
Comment (rsinnet) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:96]: It was out of date and untested so needed to be updated.

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ class LinearDispenser(_Dispenser):
     """Dispenses food over time."""
 
     _TIME_STEP = .1
-    _LBS_PER_SEC = .5
+    _LBS_PER_SEC = 1 * 454
```
Comment (zzv2) [inline miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:46]: where does this number come from, can you add a comment for future devs?

File: miso_peripherals/nodes/ram_hopper_simulator_node [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
 
 def main():
     rospy.init_node('ram_hopper_simulator_node')
-    dispenser_type = globals()[
-        rospy.get_param('~dispenser_type', 'LinearDispenser')]
 
-    controller = SimHopperController(dispenser_type)
+    # Load the selected dispenser type from the sim driver.
+    dispenser_type = getattr(
+        import_module('miso_peripherals.hopper.sim_driver'),
```
Comment (zzv2) [inline miso_peripherals/nodes/ram_hopper_simulator_node:16]: huh

**Review by zzv2 — APPROVED** [review]

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py (hunk 2 above)
Comment (rsinnet) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:46]: I made it up lol

**Comment by rsinnet** [comment]
This is confirmed fixed in sim. :shipit:

**Pull request merged**